packages/plugin-renderer/src/RenderingEngine.ts

```diff
@@ -89,12 +89,22 @@ export class RenderingEngine<T = {}> {
      * @param nodes
      * @param rendering
      */
-    async rendered(nodes: VNode[], rendering: [Renderer<T>, Promise<T>]): Promise<T> {
+    async rendered(nodes: VNode[], renderer: Renderer<T>, rendered: Promise<T>): Promise<T> {
         for (const node of nodes) {
-            const renderings = this.renderings.get(node) || [];
-            renderings.push(rendering);
+            let renderings = this.renderings.get(node);
+            if (!renderings) {
+                renderings = [];
+                this.renderings.set(node, renderings);
+            }
+            const index = renderings.findIndex(rendering => rendering[0] === renderer);
+            if (index === -1) {
+                renderings.push([renderer, rendered, nodes]);
+            } else {
+                renderings[index][1] = rendered;
+                renderings[index][2].push(...nodes);
```

Comment:
Mais non ? La valeur va être settée sur `this.renderings` pour chaque noeud donc je vois pas pourquoi il faudrait rajouter une info en plus dans la map.